test/integration-test.py

```diff
@@ -308,9 +308,21 @@ def main():
     if not startservers.check():
         raise Exception("startservers.check failed")
 
+    check_slow_queries()
+
     global exit_status
     exit_status = 0
 
+def check_slow_queries():
+    """Checks that we haven't run any slow queries during the integration test.
+
+    This depends on flags set on mysql in docker-compose.yml.
```

Comment:
small nit, the flags are being set on `mysqld` (emphasis on the trailing **d**)
```suggestion
    This depends on flags set on mysqld in docker-compose.yml.
```